editor.planx.uk/src/@planx/components/FileUploadAndLabel/Modal.tsx

```diff
@@ -267,14 +280,27 @@ const SelectMultiple = (props: SelectMultipleProps) => {
           },
         }}
       >
+        <ListSubheader disableGutters>
+          <ListHeader>
+            <Typography variant="h4" component="h3" pr={3}>
+              Select all document types that apply
```

Comment:
nit: "document types" feels like brand new language only used here - would "Select all labels that apply" better match the error messages that might be shown later on validation? If this copy came from content team/user research, obviously feel free to ignore my suggestion :slightly_smiling_face:

Reply:
Thanks @jessicamcinchak, I will ask August about this one.

Reply:
I've updated to this (Select all labels that apply) for now, will add as a separate PR if there is further feedback from content design.